rootfs/api/serializers.py

```diff
@@ -256,10 +253,11 @@ def validate_domain(self, value):
         if 'xip.io' in value:
             return value
 
+        # Let wildcards through by not trying to validate it
         if labels[0] == '*':
```

Comment:
That's fair. I addressed that by adding in a validation if no label is left after the popping, and added it to the "bad domain" test